src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    M['<uri>.params.ssl_ca_file']
+
+M['iproto.listen.*.params.ssl_cert_file'] =
+    M['<uri>.params.ssl_cert_file']
+
+M['iproto.listen.*.params.ssl_ciphers'] =
+    M['<uri>.params.ssl_ciphers']
+
+M['iproto.listen.*.params.ssl_key_file'] =
+    M['<uri>.params.ssl_key_file']
+
+M['iproto.listen.*.params.ssl_password'] =
+    M['<uri>.params.ssl_password']
+
+M['iproto.listen.*.params.ssl_password_file'] =
+    M['<uri>.params.ssl_password_file']
+
+M['iproto.listen.*.params.transport'] =
+    M['config.storage.endpoints.*.params.transport']
+
+M['iproto.listen.*.uri'] = M['iproto.listen.*']
+
+M['iproto.net_msg_max'] = format_text([[
+    To handle messages, Tarantool allocates fibers. To prevent fiber overhead
+    from affecting the whole system, Tarantool restricts how many messages the
+    fibers handle, so that some pending requests are blocked.
+
+    - On powerful systems, increase `net_msg_max`, and the scheduler starts
+    processing pending requests immediately.
+    - On weaker systems, decrease `net_msg_max`, and the overhead may decrease.
+    However, this may take some time because the scheduler must wait until
+    already-running requests finish.
+
+    When `net_msg_max` is reached, Tarantool suspends processing of incoming
+    packages until it has processed earlier messages. This is not a direct
+    restriction of the number of fibers that handle network messages, rather
+    it is a system-wide restriction of channel bandwidth. This in turn restricts
+    the number of incoming network messages that the transaction processor
+    thread handles, and therefore indirectly affects the fibers that handle
+    network messages.
+]])
+
+M['iproto.readahead'] = format_text([[
+    The size of the read-ahead buffer associated with a client connection. The
+    larger the buffer, the more memory an active connection consumes, and the
+    more requests can be read from the operating system buffer in a single
+    system call.
+
+    The recommendation is to make sure that the buffer can contain at least a
+    few dozen requests. Therefore, if a typical tuple in a request is large,
+    e.g. a few kilobytes or even megabytes, the read-ahead buffer size should
+    be increased. If batched request processing is not used, it's prudent to
+    leave this setting at its default.
+]])
+
+M['iproto.threads'] = format_text([[
+    The number of network threads. There can be unusual workloads where the
+    network thread is 100% loaded and the transaction processor thread is not,
+    so the network thread is a bottleneck. In that case, set `iproto_threads`
+    to 2 or more. The operating system kernel determines which connection goes
+    to which thread.
+]])
+
+-- }}} iproto configuration
+
+-- {{{ labels configuration
+
+M['labels'] = format_text([[
+    The `labels` section allows adding custom attributes to the configuration.
+    Attributes must be key: value pairs with string keys and values.
+]])
```

Comment:
The colon here is interpreted as a punctuation mark by my brain: I spin here for a while to understand '<...> key: value pairs' as 'key-value pairs'.

I would just say 'the keys and values are strings'.

Reply:
Ok. Let's do it this way.